src/utils/astropy_util.py

```diff
@@ -0,0 +1,30 @@
+from astropy.time import Time
+from astropy.coordinates import get_sun, get_moon, CartesianRepresentation
+from astropy.constants import G, M_sun, M_earth
+from typing import Tuple, Optional
+from utils.constants import BodyEnum
+from astropy.coordinates import SkyCoord
+
+
+# Heavily reference the get_ephemeris function here: https://github.com/Cislunar-Explorers/FlightSoftware/blob/master/OpticalNavigation/core/observe_functions.py
+def get_ephemeris(observeStart: float, body: BodyEnum) -> Tuple[float, float, float]:
```

Comment:
Rename `observeStart` to something more applicable to the purposes of your use case. Just because you're referencing my code doesn't mean you need to keep all the variable names the same